dp/training/metrics.py

```diff
@@ -3,10 +3,38 @@
 
 
 def word_error(predicted: List[Union[str, int]], target: List[Union[str, int]]) -> float:
+    """Calculates the word error rate of a single word result.
+
+    Args:
+      predicted: Predicted word.
+      target: Target word.
+      predicted: List[Union[str: 
```

Comment:
Parameter annotation already done in function signature. Not needed in the docstring.